typescript-client-sdk/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+  "semi": true,
+  "trailingComma": "none",
+  "singleQuote": false,
+  "printWidth": 123
```

Comment:
```suggestion
  "printWidth": 120
```

Reply:
Yeah we can go with 120. 123 was what caused the least amount of changes to the file as it is.

Reply:
Fixed. But also lol those were my changes anyway. Never too soon to add Prettier 😂